mmdet/core/evaluation/text_evaluation.py

```diff
@@ -284,8 +287,8 @@ def match(gt_polygons_list, gt_transcriptions, gt_dont_care_polygon_nums,
                         if  match_transcriptions(gt_transcriptions[gt_idx].lower(), pr_transcriptions[pr_idx].lower(), word_spotting):
                             pr_matched_nums.append(pr_idx)
                         else:
-                            print(gt_transcriptions[gt_idx],
-                                  pr_transcriptions[pr_idx])
+                            # print(gt_transcriptions[gt_idx],
+                            #       pr_transcriptions[pr_idx])
```

Comment:
Is it safe to remove the comment as well?

Reply:
removed